### PR TITLE
⬆️ release 1.0.1

### LINKED ISSUE #21: Add button link to open from preferencs page
Add preferences to the extension.
On click of preferences/options , the `know your permissions` page should open

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #21

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,5 +26,11 @@
     "jszip": "^3.1.5",
     "ncp": "^2.0.0",
     "rimraf": "^2.6.2"
+  },
+  "options_ui": {
+    "page": "src/index.html",
+    "browser_style": false,
```
Comment (tsl143) [inline package.json]: remove this, not compatible for chrom
chrome_style works for both

Comment (championshuttler) [inline package.json]:
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -26,5 +26,10 @@
     "jszip": "^3.1.5",
     "ncp": "^2.0.0",
     "rimraf": "^2.6.2"
+  },
+  "options_ui": {
+    "page": "src/index.html",
```
Comment (tsl143) [inline package.json]: path should be `index.html`.
Build addon will not include `src` folder :P

Comment (championshuttler) [inline package.json]:
```suggestion
    "page": "index.html",
```

**Pull request merged**